topydo/ui/columns/Main.py

```diff
@@ -43,6 +44,60 @@
 from topydo.ui.columns.ColumnLayout import columns
 
 
+class Transaction(object):
+    """
+    This class implements basic handling of executing any subcommand on multiple
+    todo items.
+    """
+    def __init__(self, p_todo_ids=None):
+        self._subcommand = None
+        self._todo_ids = p_todo_ids
+        self._operations = []
+
+    def prepare(self, p_subcommand, p_args):
```

Comment:
This parameter `p_subcommand` should not be here I think. A transaction is a sequence of the _same_ commands, I don't think it's necessary to pass it as a parameter of `prepare()`. Since the command to-be executed is a property of the transaction, I'd expect you'd pass it on as part of the constructor.